config/initializers/new_framework_defaults_5_1.rb

```diff
@@ -10,8 +10,7 @@
 Rails.application.config.action_view.form_with_generates_remote_forms = false
 
 # The belongs_to associations are required by default in Rails 5
-# we cannot opt into this right now, but it would be a goal for the future
-Rails.application.config.active_record.belongs_to_required_by_default = false
+Rails.application.config.active_record.belongs_to_required_by_default = true
```

Comment:
🎉

Reply:
Did not know that, nice catch!

Reply:
Only if we actually enable the 5.1 defaults 😉 But I guess we can be save to do that.